pyproject.toml

```
@@ -0,0 +1,109 @@
+[tool.poetry]
+name = "django-easy-audit"
+version = "1.3.6-a2"
```

Probably not important for now but consistent:

```suggestion
version = "1.3.6"
```